pom.xml

```diff
@@ -13,10 +14,20 @@
     <url>http://wiki.jenkins-ci.org/display/JENKINS/DTKit</url>
 
     <properties>
-        <java.level>8</java.level>
         <dtkit-frmk.version>3.0.0</dtkit-frmk.version>
-        <jenkins.version>2.319.1</jenkins.version>
+        <jenkins.version>2.346.3</jenkins.version>
```

Comment:
this is quite an API plugin and must match what xunit refers, so upgrade jenkins version means force also xunit plugin to be updated.

Reply:
Thanks for the reply! Could you elaborate on what changes you'd like to see?

I'm currently running this plugin on Jenkins 2.402.1